config-templates/ibm/ess/2.1.0/custom-parameters.json

```diff
@@ -0,0 +1,172 @@
+[
+	{
+		"description": "The mount path of the primary file system.",
+		"displayname": "Scale Host Path",
+		"name": "scale-host-path",
+		"required": "true",
+		"type": ""
+	},
+	{
+		"description": "The cluster ID of the pirmary IBM Spectrum Scale cluster.",
+		"displayname": "Cluster ID",
+		"name": "cluster-id",
+		"required": "true",
+		"regex": "^[0-9]+$",
+		"type": ""
```

Comment:
Either remove `"type": ""` or explicitly set it to `text`. 
Similarly for other parameters `primary-fs`, `gui-host`, `secret-name`, `k8-n1-ip`, `sc-n1-host`, `k8-n2-ip`, `sc-n2-host`,  `k8-n3-ip`, `sc-n3-host`, `k8-n4-ip`, `sc-n4-host`, `k8-n5-ip`, `sc-n5-host`, `k8-n6-ip`, `sc-n6-host`, `k8-n8-ip`, `sc-n7-host`, `k8-n8-ip`, `sc-n8-host`,